src/Slider.jsx

```diff
@@ -335,11 +348,15 @@ export default React.createClass({
       document.attachEvent('onmouseup', this.endSlide);
     }
 
-    return this.killEvent(ev);
```

Comment:
There's a `return;` higher up in the function - any return statement means that every exit point must have a return statement.